certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json

```diff
@@ -423,7 +423,7 @@
                         "title": "Webpage"
                       },
                       "Status": {
-                        "type": "string",
+                        "type": "integer",
```

Comment:
These input fields "type", has been changed from string to integer. can you please confirm that no existing flows or users will be affected by this change?